### ISSUE TITLE
brew formula not supported MacOS 

### ISSUE THREAD

**Issue by bodhibyte** [archiecobbs/s3backer#225]
Hello,

I'm trying to install s3backer on MacOS Ventura, and I'm getting the following error. I checked the https://formulae.brew.sh/formula/s3-backer page, and it says "Requires: Linux".

% brew install s3-backer
==> Downloading https://formulae.brew.sh/api/formula.jws.json
######################################################################### 100.0%
==> Downloading https://formulae.brew.sh/api/cask.jws.json
######################################################################### 100.0%
s3-backer: Linux is required for this software.
libfuse@2: Linux is required for this software.
Error: s3-backer: Unsatisfied requirements failed this build.

**Comment by archiecobbs** [comment]
Hmm. I'm not that familiar with brew build scripts.

Where is it getting "Requires Linux" from?

I'm not saying I don't believe it, I'm just trying to understand what's causing that determination by brew. After all, this used to work once.

Thanks.

**Comment by davidsmith91** [comment]
> Hmm. I'm not that familiar with brew build scripts.
> 
> Where is it getting "Requires Linux" from?
> 
> I'm not saying I don't believe it, I'm just trying to understand what's causing that determination by brew. After all, this used to work once.
> 
> Thanks.

You can just see that on homebrew website

<img width="737" height="675" alt="Image" src="https://github.com/user-attachments/assets/55d5694b-63cb-4aae-a567-023cc5e9f39a" />

Anyway, I'm trying to setup a way to have a "infinite" storage that is "automatically" backed up (being on cloud), which can be used between multiple devices of multiple people that are connected to the same VPN (with DERP server close). But the data will be 8000km far. But I want to be able to have a cache for the interesting data (the recent used ones) and browse the files fast (like a cache of metadata that is "close", not 8000km far).

So let's say: tomorrow I'll be in Thailand or in Australia or Russia or South America etc.
I want to have the main data in Europe. But be able of using them anywhere together with other people.

So I was looking for something that can actually mimic a real filesystem, with the locking mechanisms and real "working" mechanisms. 

I tried juicefs some time ago and it was too slow unless I had the data near me, even if I was having the metadata near me. That didn't make sense to me: isn't the metadata separation done with the intent of being able to have it close and avoid slow loading times?

I want to mount the files on every device I have: android phones + all the main desktop operating systems (macos, linux, windows).

On android there are open source apps to mount various kind of storage. Webdav, samba, etc. And then it is available to the Files app and other apps.

And on desktop operating systems there are ways to mount filesystems too. 

But I tried different things: rclone mount, webdav, juicefs etc. Always having some kind of problem either with syncronization, or file locking or really unusable slow speeds

Basically, what I would want in theory, is the possibility to have the data in Europe while being 8000km away (for example), but having a cache of recent used files, and/or the metadata near me, so I can search fast. Then, if I want to transfer something, like downloading (reading) or uploading (writing) I will just know that if it's a video or an archive, it will take some time more etc. But like, it should work even on 4g for most things. I'm okay with waiting, but from a bandwidth point of view, not with thousands of micro operationg having latency. Is something like this possible?

And how? Can s3backer be used for this?

**Comment by archiecobbs** [comment]
Just to be clear, s3backer doesn't support simultaneous mounting of the same S3 data by more than one machine at the same time. So "together with other people" and "mount the files on every device I have" at the same time aren't options with s3backer.

Sounds like what you want is more of a [Global file system](https://en.wikipedia.org/wiki/Global_file_system)....
